nemo_rl/models/policy/megatron_policy_worker.py

```
@@ -1782,40 +1781,120 @@ def generate(
+            DynamicInferenceEngine,
+        )
         from megatron.core.inference.model_inference_wrappers.gpt.gpt_inference_wrapper import (
             GPTInferenceWrapper,
         )
-
-        inference_context = StaticInferenceContext.from_config(inference_wrapper_config)
-
+        from megatron.core.inference.sampling_params import SamplingParams
+
+        inference_max_batch_size = inference_wrapper_config.inference_max_requests
+        max_batch_size = inference_max_batch_size
```

Ruff (0.14.4)

Local variable `max_batch_size` is assigned to but never used

Remove assignment to unused variable `max_batch_size`

(F841)